backends-clickhouse/src-celeborn/main/scala/org/apache/spark/shuffle/CHCelebornColumnarShuffleWriter.scala

```diff
@@ -80,10 +79,10 @@ class CHCelebornColumnarShuffleWriter[K, V](
       nativeBufferSize,
       capitalizedCompressionCodec,
       compressionLevel,
-      GlutenConfig.get.chColumnarShuffleSpillThreshold,
```

Comment:
Can we keep the use of `GlutenConfig.get` after the refactor? I prefer a uniform API. It requires either velox conf or CH conf to be covered by GlutenConfig at runtime somehow.

Reply:
So that is to say, the definition of the configuration is independent, but all the acquisitions are from GlutenConfig?

Reply:
Perhaps we can make `VeloxConfig` inherit from `GlutenConfig`? Then `VeloxConfig` can be used uniformly in Velox backend for both Velox / Common configurations.

Reply:
The current implementation is an inheritance relationship.

Reply:
Aha, then I feel it's already convenient to developers.

Reply:
> So that is to say, the definition of the configuration is independent, but all the acquisitions are from GlutenConfig?

@yikf, yes, it's not a mature idea. Maybe, firstly keep using your current changes.